ims/common/config.py

```diff
@@ -9,53 +9,94 @@
 __config = None
```

Comment:
So that only one config object exists in the entire application. We dont want to keep loading a different object in each module.